src/resolver/resolver.zig

```diff
@@ -2612,6 +2613,7 @@ pub const Resolver = struct {
         return try r.dirInfoCachedMaybeLog(path, false, true);
     }
 
+    /// Like `readDirInfo`, but returns `null` instead of throwing an error.
```

Comment:
Was this comment helpful for you? The lack of a `!` and the name communicate this information as well

Reply:
It makes finding related methods easier when hovering over usages in my editor